ComponentKit.xcodeproj/project.pbxproj

```diff
@@ -3404,10 +3404,6 @@
 		B342DC471AC23E8200ACAC53 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				FRAMEWORK_SEARCH_PATHS = (
-					"$(inherited)",
-					"$(PROJECT_DIR)/Carthage/Build/iOS",
-				);
```

Comment:
Yes – they are in a separate commit to fix a build warning. Can move to another PR if that makes more sense.